samples/perf_sgemm.cpp

```diff
@@ -671,7 +671,7 @@ ROCWMMA_HOST void gemm_test(uint32_t m, uint32_t n, uint32_t k, float32_t alpha,
     if((m < get<0>(macroTileSize) || n < get<1>(macroTileSize) || k < ROCWMMA_K)
        || (m % ROCWMMA_M || n % ROCWMMA_N || k % ROCWMMA_K))
     {
-        std::cout << "Unsupported size!\n";
+        std::cout << "Unsupported matrix size!\n";
         return;
     }
 
```

Comment:
Need to do similar checks as above, except sgemm isn't valid on gfx11.
    
    if(isGfx11() )
    {
        std::cout << "Unsupported architecture!\n";
        return;
    }
   ...
   // check block and wave size for gfx9... etc.
   ...